met-api/src/met_api/schemas/comment.py

```diff
@@ -24,5 +24,22 @@ class Meta:  # pylint: disable=too-few-public-methods
     review_date = fields.Str(data_key='review_date')
     status_id = fields.Int(data_key='status_id')
     survey_id = fields.Int(data_key='survey_id')
+    submission_id = fields.Int(data_key='submission_id')
     comment_status = fields.Nested(CommentStatusSchema)
     survey = fields.Pluck(SurveySchema, 'name')
+    question = fields.Method('get_comment_question')
+
+    def get_comment_question(self, obj):
+        """Get the associated question of the comment."""
+        components = list(obj.survey.form_json.get('components', []))
+        if len(components) == 0:
+            return None
+        component_label = [
+            component.get(
+                'label',
+                None) for component in components if component.get(
+                'key',
+                None) == obj.component_id]
+        if len(component_label) == 0:
+            return None
+        return component_label[0]
```

Comment:
i am just thinking aloud , does it help to store the question as such in comments table? i know its duplication and eats up space , but added value is performance and we can remove this logic...what do you think?
Its not ideal i know , just hate the hassle of going through the compoent json to extract the question

Reply:
Yep I thought about that, also there's the versioning of the question (lets say someone changes the question text in the future) this would keep exctly what was asked at the time of the comment creation, that said the only reason I stored the id instead of the question is bc of duplication, but if we think its enough value i'm happy to store it too